deployments/rds-s3/terraform/main.tf

```diff
@@ -159,9 +159,6 @@ module "eks_blueprints_kubernetes_addons" {
   enable_aws_load_balancer_controller = true
   enable_cert_manager                 = true
 
-  cert_manager = {
```

Comment:
why are we unpinning cert manager?

Reply:
Ran into a couple flakey issues with cert manager. Will pin again and run continuous testing to see how flakey it is.

Reply:
reverted, will run continuous tests before releasing